MAINTAINERS.md

```diff
@@ -81,6 +81,10 @@ We use the following workflow for patch releases:
         commit tag:"6.0.0" id:"latest"
 ```
 
+## Consider if a deprecation warning needs a change request
+
+Sometimes we may get deprecation warnings, but we might not always want to resolve this with a change request. This is because these changes can be breaking changes for the consumers of our library. Instead, you can suppress the deprecation by adding the name of the deprecation to the `silenceDeprecations` list and documenting the suppression.
```

Comment:
```suggestion
Sometimes we may get deprecation warnings, but we might not always want to resolve this with a change request. This is because these changes can be breaking changes for the consumers of our library who reuse our uncompiled sass. The largest group of which are prototypes. Instead, you can suppress the deprecation by adding the name of the deprecation to the `silenceDeprecations` list and documenting the suppression.
```

Reply:
in my head I'm thinking it would be easy to think that because we compile our sass to css, that deprecations won't be breaking. And the integration of our styles with prototypes might be non-obvious for future people joining.